--- origsrc/sox-14.1.0/src/Makefile.am	2008-07-23 19:53:05.000000000 -0500
+++ src/sox-14.1.0/src/Makefile.am	2008-08-04 14:59:16.687500000 -0500
@@ -242,6 +242,7 @@
 endif
 
 libsfx_la_CFLAGS = @SAMPLERATE_CFLAGS@
+libsfx_la_LDFLAGS = -no-undefined
 libsfx_la_LIBADD = @PNG_LIBS@ @SAMPLERATE_LIBS@ libsox.la
 
 ############################################
@@ -254,6 +255,7 @@
 	  xmalloc.c xmalloc.h getopt.c getopt1.c getopt.h \
 	  soxconfig.h util.c util.h libsox.c libsox_i.c
 libsox_la_CFLAGS = 
+libsox_la_LDFLAGS = -no-undefined
 
 if HAVE_LIBLTDL
 else
