--- origsrc/sox-14.4.0/src/Makefile.am	2012-03-04 16:54:16.000000000 -0600
+++ src/sox-14.4.0/src/Makefile.am	2012-10-25 23:42:53.159102300 -0500
@@ -12,7 +12,7 @@ if HAVE_LIBLTDL
 # is ignored when you specify a more specific one.
 # We want to version libsox and we are OK because they
 # have a more specific LDFLAGS that includes -version-info.
-AM_LDFLAGS = -avoid-version -module
+AM_LDFLAGS = -avoid-version -module @APP_LDFLAGS@
 AM_CPPFLAGS += -DPKGLIBDIR="\"$(pkglibdir)\""
 endif
 if HAVE_WIN32_LTDL
